tests/subsys/pm/power_mgmt_soc/boards/mimxrt595_evk_cm33.conf

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright 2022, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_REGULATOR=y
+# Turn on Device Level Power Management as we wish
+# to reconfigure the FlexSPI pins for power savings
+# when transitioning the SoC to Deep Low Power modes.
+CONFIG_PM_DEVICE=y
```

Comment:
Why is `CONFIG_PM_DEVICE` required here, but not in the board specific system off sample?

Reply:
Thanks for catching this, I have added to the sample as well.